rllib/utils/pre_checks/env.py

```diff
@@ -30,12 +30,14 @@ def check_env(env: EnvType) -> None:
             "Env must be one of the supported types: BaseEnv, gym.Env, "
             "MultiAgentEnv, VectorEnv, RemoteBaseEnv")
 
-    if isinstance(env, MultiAgentEnv):
-        check_multiagent_environments(env)
-    elif isinstance(env, gym.Env):
+    if isinstance(env, gym.Env) and not isinstance(env, MultiAgentEnv):
         check_gym_environments(env)
-    elif isinstance(env, VectorEnv) or isinstance(env, RemoteBaseEnv):
-        logger.warning("Env checking isn't implemented fro VectorEnvs or "
+    elif isinstance(env, MultiAgentEnv):
+        check_multiagent_environments(env)
```

Comment:
Sorry to pick this up again, but isn't:

```
if isinstance(env, MultiAgentEnv):
    check_multiagent_environments(env)
elif isinstance(env, gym.Env):
    check_gym_environments(env)
```

the same and easier to read?

Reply:
whoops -- will fix :)